tools/microsats_alignment_level/microsats_alignment_level.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="microsats_align1" name="Extract Orthologous Microsatellites" version="1.0.0">
+<tool id="microsats_align1" name="Extract Orthologous Microsatellites" version="1.0.1">
   <description> from pair-wise alignments</description>
   <requirements>
     <requirement type="package" version="1.0">sputnik</requirement>
```

Comment:
There is a sputnik on conda but it's a different software (this is one has been dead for a long time).

Reply:
And one of the tarballs to build it was on bitbucket, I'll just remove this tool from the PR.